src/ui/components/Rating/styles.scss

```diff
@@ -71,52 +70,19 @@
 
 .Rating--editable {
   .Rating-star {
-    background-image: url('./img/open-star-dim-yellow.svg');
     cursor: pointer;
   }
 
   &.Rating.Rating--loading .Rating-star {
     animation-name: pulseOpaqueRatingStars;
     // Ensure editable rating stars always render as open stars while
     // loading, even if some are selected (e.g. while hovering).
-    background-image: url('./img/open-star-dim-yellow.svg');
     cursor: initial;
   }
 }
 
-.Rating-selected-star {
-  background-image: url('./img/closed-star-gray.svg');
-}
-
-.Rating--editable .Rating-selected-star {
-  background-image: url('./img/closed-star-yellow.svg');
-}
-
 .Rating-half-star {
-  background-image: url('./img/half-star-gray.svg');
-
   [dir='rtl'] & {
     transform: scaleX(-1);
   }
 }
-
-.Rating--yellowStars {
```

Comment:
It looks like this can be deleted now:

https://github.com/mozilla/addons-frontend/blob/c258fabf592ac0e321d750d4bf414b9c968eb97d/src/ui/components/Rating/index.js#L218